recipes/malt/meta.yaml

```diff
@@ -0,0 +1,32 @@
+# This file created by conda-build 3.15.1
+# meta.yaml template originally from:
+# /opt/recipe, last modified Fri Oct 26 10:28:41 2018
+# ------------------------------------------------
+
+package:
+    name: malt
+    version: 0.41
+source:
+    md5: 1e7b3d126e3f761e7d09ddf0029a3dff
+    url: https://ab.inf.uni-tuebingen.de/data/software/malt/download/MALT_unix_0_4_1.sh
+build:
+    noarch: generic
+    number: '0'
+    string: '0'
+requirements:
+    build:
+        - openjdk 8.0.144 zulu8.23.0.3_2
```

Comment:
Does this actually need openJDK to install?

Reply:
Well it needs Java 8 specifically and will fail with Java 9 or higher. or am I misunderstanding the question?

Reply:
The question is whether the installation script needs java in order to run.

Reply:
Ah I see, no the installation script does not need a specific java version to run as far as i know.  So I guess I can remove that line

also removing
build:
    noarch: generic
    number: '0'
    string: '0'

resulted in following error message
ERROR: recipes/malt/meta.yaml:1: missing_build_number: The recipe is missing a build number
ERROR: recipes/malt/meta.yaml:1: should_be_noarch_generic: The recipe should be build as `noarch`

Reply:
Turns out the script really needs openjdk to run after all

Reply:
Fair enough